web/sites/default/config/search_api.server.searchstax.yml

```diff
@@ -34,17 +34,17 @@ backend_config:
   connector: basic_auth
   connector_config:
     scheme: https
-    host: 'null'
+    host: ss626374-s4p42op1-us-west-2-aws.searchstax.com
     port: 443
     path: /
-    core: 'null'
+    core: Test
     timeout: 5
     index_timeout: 5
     optimize_timeout: 10
     finalize_timeout: 30
     commit_within: 1000
-    username: 'null'
-    password: 'null'
+    username: Drupal
+    password: 'Zw92%C*VcxMo'
```

Comment:
Hmm. I don't think we want a secret in our repo 😳

Reply:
Hmmm, thank you for catching that. It seems updating the module overwrote the keys setting.